SPECS/util-linux/util-linux.spec

```diff
@@ -101,9 +103,10 @@ rm -rf %{buildroot}/lib/systemd/system
 %{_mandir}/man3/*
 
 %changelog
-* Sat May 09 00:20:52 PST 2020 Nick Samson <nisamson@microsoft.com>
-- Added %%license line automatically
-
+*   Fri Sep 04 2020 Daniel Burgener <daburgen@microsoft.com> 2.32.1-4
+-   Enable SELinux support
+*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com>
```

Comment:
><nisamson@microsoft.com> [](start = 32, length = 24)

Since you're here: could you add the version info to this line? It seems we're missing "2.32.1-3".

Reply:
Yep, done.